requirements.txt

```diff
@@ -0,0 +1,6 @@
+cffi==1.14.5
```

Comment:
This is not a direct dependency, but I confirmed that it is pulled in by `cryptography`, so I'm fine with it being pinned here.